### DISCUSSION TITLE
Can't import pysr module successfully in jupyter

### DISCUSSION THREAD

**Discussion by RicardoSun** [MilesCranmer/PySR#382] [Q&A]
I have already installed 'pysr' with pip command in vscode. And I can find it with 'pip list' and 'pip show pysr'. But when I try to 'import pysr' in a  .ipynb file, it always make an error:  No module named 'pysr'. My OS is Win10.  Thanks for your kind suggestion for this problem!

**Comment by MilesCranmer** [comment]
Hi @RicardoSun,

You are likely using a different Python environment in jupyter, where PySR is not installed. You can always do

```
!pip install -U pysr
```

and

```
import pysr
pysr.install()
```

from within Jupyter.

Cheers,
Miles

**Comment by EricLou1029** [comment]
i have the same issue here, still cant fix this

**Reply by MilesCranmer** [reply]
I think you are likely in the wrong environment?
